cartography/models/azure/cosmosdb/sqldatabase.py

```diff
@@ -0,0 +1,76 @@
+from dataclasses import dataclass
+
+from cartography.models.core.common import PropertyRef
+from cartography.models.core.nodes import CartographyNodeProperties
+from cartography.models.core.nodes import CartographyNodeSchema
+from cartography.models.core.relationships import CartographyRelProperties
+from cartography.models.core.relationships import CartographyRelSchema
+from cartography.models.core.relationships import LinkDirection
+from cartography.models.core.relationships import make_target_node_matcher
+from cartography.models.core.relationships import OtherRelationships
+from cartography.models.core.relationships import TargetNodeMatcher
+
+
+@dataclass(frozen=True)
+class AzureCosmosDBSqlDatabaseProperties(CartographyNodeProperties):
+    id: PropertyRef = PropertyRef("id")
+    lastupdated: PropertyRef = PropertyRef("lastupdated", set_in_kwargs=True)
+    location: PropertyRef = PropertyRef("location")
+    name: PropertyRef = PropertyRef("name")
+    type: PropertyRef = PropertyRef("type")
+    throughput: PropertyRef = PropertyRef("throughput")
+    maxthroughput: PropertyRef = PropertyRef("maxthroughput")
```

Comment:
The old cypher query had these as "options.throughput" and "options.autoscale_setting.max_throughput" respectively